hack/build-local-images.py

```diff
@@ -138,12 +149,11 @@ def image_rebuild_requested(image):
     suffix explicitly or does not provide
     any explicit requests.
     """
-    return len(sys.argv) == 1 or (
```

Comment:
wouldn't the diff to use a default just be:

```diff
-    return len(sys.argv) == 1 or (
+    return (len(sys.argv) == 1 and image_config[image].get("enable_default", True)) or (
```

Reply:
Yes, do you want me to change this back?

Reply:
Yes please